### PR TITLE
Add myself as the PS4/PS5 code owner

### PR DESCRIPTION
**Pull request by jmorse**
As discussed here: https://discourse.llvm.org/t/new-ps4-ps5-code-owner/80901

### CODE REVIEW

File: llvm/CODE_OWNERS.TXT [hunk 1]
```diff
@@ -196,9 +196,9 @@ N: Philip Reames
 E: listmail@philipreames.com
 D: IndVar Simplify, Scalar Evolution
 
-N: Paul Robinson
-E: paul.robinson@sony.com
-D: Sony PlayStation®4 support
+N: Jeremy Morse
+E: jeremy.morse@sony.com
+D: Sony PlayStation®4 and PlayStation®5 support
```
Comment (pogo59) [inline llvm/CODE_OWNERS.TXT:201]: The entries should be sorted by surname, please move the new entry.

**Comment by jmorse** [comment]
Switched to the new Maintainers.md format; it no longer appears to be surname ordered, instead PS things land in "other".

**Review by pogo59 — APPROVED** [review]
LGTM, although probably there should also be an entry in clang/CodeOwners.rst, as we--uh, Sony has a bunch of code sprinkled through Clang (driver target, lots of fiddly bits here and there, and a special case almost everywhere that there's an ABI difference to decide on).

**Comment by jmorse** [comment]
For the clang Maintainers.md, I figure it's worth chatting to folks at the conference to see whether that's seen as worthwhile (I don't see other target-specific entries there).

**Pull request merged**